org.eclipse.gef/src/org/eclipse/gef/ui/parts/GraphicalViewerKeyHandler.java

```diff
@@ -172,16 +170,16 @@ private <T extends EditPart> List<T> getValidNavigationTargets(List<T> list) {
 	 * @param exclude   The EditPart to be excluded from the search
 	 *
 	 */
-	GraphicalEditPart findSibling(List siblings, Point pStart, int direction, EditPart exclude) {
+	GraphicalEditPart findSibling(List<? extends GraphicalEditPart> siblings, Point pStart, int direction,
+			EditPart exclude) {
 		GraphicalEditPart epCurrent;
 		GraphicalEditPart epFinal = null;
 		IFigure figure;
 		Point pCurrent;
 		int distance = Integer.MAX_VALUE;
 
-		Iterator iter = getValidNavigationTargets(siblings).iterator();
-		while (iter.hasNext()) {
-			epCurrent = (GraphicalEditPart) iter.next();
+		for (GraphicalEditPart element : getValidNavigationTargets(siblings)) {
+			epCurrent = element;
```

Comment:
I don't think it is necessary anymore to store `element` in a local variable.

Reply:
Thx for noting. Fixed!